.vscode/extensions.json

```
@@ -0,0 +1,6 @@
+{
+  "recommendations": [
+    "puppet.puppet-vscode",
+    "rebornix.Ruby"
```

Can we add `ms-vscode-remote.vscode-remote-extensionpack` here?
```suggestion
    "rebornix.Ruby",
    "ms-vscode-remote.vscode-remote-extensionpack"
```

---

This file is also from the pdk template: https://github.com/puppetlabs/pdk-templates/blob/main/moduleroot/.vscode/extensions.json

It doesn't seems to be a way to change that via any variables. But we can just set this file as 'unmanaged' by PDK and then write into it whatever we want.

I don't use vscode, so any suggestions for people using it is welcome.

---

OK, maybe that could become an upstream issue/PR.